src/bioregistry/data/bioregistry.json

```diff
@@ -89697,7 +89697,7 @@
           "description": "Plant Ontology through Amigo",
           "homepage": "http://www.plantontology.org/",
           "name": "Plant Ontology through Amigo",
-          "uri_format": "http://www.plantontology.org/amigo/go.cgi?view=details&query=PO:$1"
+          "uri_format": "https://browser.planteome.org/amigo/term/PO:$1"
```

Comment:
This instance of the URL is ingested from identifiers.org (it is under the `miriam` section) so it isn't meant to be manually updated.

Reply:
Thank you for the prompt feedbak!

<img width="936" alt="Screenshot 2024-08-06 at 14 14 10" src="https://github.com/user-attachments/assets/0d1cbd7e-a42b-4454-9729-50e662b94ac7">

How can we change this then?

Reply:
Oh, I see now, sorry my bad, just did a ctrl+F too quickly